app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -400,11 +369,12 @@ class GithubWebhook extends RequestHandler<Body> {
   }
 
   Future<void> _checkForGoldenTriage(
-    PullRequestEvent event,
+    String eventAction,
+    PullRequest event,
     List<IssueLabel> labels,
   ) async {
-    if (event.repository.fullName.toLowerCase() == 'flutter/flutter' &&
-        await _isIgnoredForGold(event)) {
+    if (needsCQLabelList.contains(event.head.repo.fullName.toLowerCase()) &&
```

Comment:
This logic is different from the previous one, actually in a few days we will remove the requirement for explicitly adding the CQ+1 label on sdk. I'd recommend created a top level constant for this use case.